.github/workflows/binaries.yml

```diff
@@ -0,0 +1,39 @@
+name: Build binaries
+on:
+  release:
+    types: [created]
```

Comment:
Should this be "published" instead?

Reply:
Yes, will change. I took it from [here](https://github.com/haskell/haskell-language-server/blob/e8bdd77921576f857368848480f131f38286c81c/.github/workflows/build.yml#L9), but e.g [this SO answer](https://stackoverflow.com/a/61066906) suggests to use `published`, but I can't find official documentation for this.